ChineseFilter/sections/specific.txt

```diff
@@ -1674,6 +1674,7 @@ iyf.tv##vg-player.ng-star-inserted > vg-pause-ads + div.caption.show > .inner
 ! https://github.com/AdguardTeam/AdguardFilters/issues/40396
 !+ PLATFORM(ios, ext_android_cb, ext_safari)
 m.xigua110.com##.ui-detail-info > div[style*="line-height:35px; text-align:center;"]
+||qidian.com^*probev3.js
```

Comment:
Replied in https://github.com/AdguardTeam/AdguardFilters/issues/178228

Not sure about the rule, but the correct place for a new rule in this file - https://github.com/AdguardTeam/AdguardFilters/blob/083d8dcadaf39bc8faa00799e9b389df4e09fab7/ChineseFilter/sections/specific.txt#L34

Reply:
Sorry for this misplace. :rofl:  I will fix this if the rule can be accepted.